File: Old/RepeatPhrase.java
import java.util.Scanner;

// RepeatPhrase.java

public class RepeatPhrase {
	public static void main(String[] args) {
		// Scanner for User Input
		Scanner input = new Scanner(System.in);
		// Input Phrase from the User
		System.out.print("Enter A Phrase: ");
		String phrase = input.nextLine();

		// Input How Many Times User Wants to Repeat It
		System.out.print("Enter The Times To Repeat: ");
		int repeat = input.nextInt();

		// Repeat the Phrase Using a Loop
		for (int i = 0; i < repeat; i++) {
			System.out.print(phrase + " ");
		}
		
		// Clean Up
		input.close();
	}
}

File: Old/RepeatPhrase.cpp
// RepeatPhrase.cpp

#include <iostream>
#include <string>
using namespace std;

int main() {

	// Input Phrase from the User
	cout << "Enter A Phrase: ";
	string phrase;
	getline(cin, phrase);

	// Input How Many Times User Wants to Repeat It
	cout << "Enter The Times To Repeat: ";
	int repeat;
	cin >> repeat;

	// Repeat the Phrase Using a Loop
	for (int i = 0; i < repeat; i++) {
		cout << phrase << " ";
	}

	return 0;
}
